### DISCUSSION TITLE
How to reduce the size of diamond.daa file

### DISCUSSION THREAD

**Discussion by vinoth-maker** [bbuchfink/diamond#834] [Q&A]
Hi! I started running the Diamond analysis after downloading the database and index file. I have a sample of paired-end reads with about 14 million reads, totaling 6 GB. After the Diamond analysis, I ended up with a 30 GB .daa file. Is this normal? How can I reduce the file size by adjusting parameters? I need to process multiple samples, and I'm concerned that the size of the file might be an issue when opening it in MEGAN. I used the following command for the analysis:  
nohup diamond blastx \
    -d /home/diamond_db/nr \
    -q /home/anacmj_alunos/vino/test/trimming/E043_trimmed.fastq \
    -o /home/anacmj_alunos/vino/test/Diamond/E043_trimmed.fastq.daa \
    -f 100 \
    --threads 20 &

Regards
Vino

**Comment by bbuchfink** [comment]
It is normal as the daa format is not very space efficient. You can try adjusting `-k` or use `--top` instead.
